lib/api/CDataFrameBoostedTreeRunner.cc

```diff
@@ -23,7 +23,8 @@ namespace ml {
 namespace api {
 namespace {
 // Configuration
-const std::string DEPENDENT_VARIABLE{"dependent_variable"};
+const std::string DEPENDENT_VARIABLE_NAME{"dependent_variable"};
+const std::string PREDICTION_COLUMN_NAME{"prediction_column_name"};
```

Comment:
As this is user facing, I think we might want to avoid the use of `column`. How about `prediction_field_name`?

Reply:
Ok, good suggested alternative. Done in [4a92476](https://github.com/elastic/ml-cpp/pull/587/commits/4a924768058c9c92625345135c4d18beb5ad0000).